CHANGES.md

```diff
@@ -1,6 +1,6 @@
 # Changes
 
-## Unreleased
+## 0.27.1
 
 - Bump geo-types
 - Allow typeck to infer pointer types
```

Comment:
```suggestion
- FIX: `network` feature fails to compile on aarch64/arm64.
```

The actual change is kind of inscrutable - probably it's more useful to explain why you did it.